ifcbdb/dashboard/management/commands/syncdataset.py

```diff
@@ -30,4 +30,4 @@ def handle(self, *args, **options):
             self.stderr.write('No such dataset "{}"'.format(dataset_name))
             return
         acc = Accession(d, lat=lat, lon=lon, depth=depth, newest_only=newest_only)
-        acc.sync(progress_callback=print, log_callback=print)
+        acc.sync(progress_callback=lambda _: True, log_callback=print)
```

Comment:
Still doesn't print stats at the end. Let me know if I am missing something here. Thanks.

Reply:
Printing stats at the end is accomplished with the progress callback, which we've modified to do nothing but return True.

You could capture the return value in `progress_callback` and retain it so that you could print a summary of it at the end.